app/file.go

```diff
@@ -1092,6 +1092,10 @@ func (a *App) GetFileInfo(fileId string) (*model.FileInfo, *model.AppError) {
 	return a.Srv.Store.FileInfo().Get(fileId)
 }
 
+func (a *App) GetFileInfos(page, perPage int, opt *model.GetFileInfosOptions) ([]*model.FileInfo, *model.AppError) {
```

Comment:
One comment on the API design: What does the API return in `nil` as options is provided?

Reply:
FileInfos that are not deleted, sorted in ascending order of date created.

Reply:
Thanks for the answer @ashishbhate. That should be sufficed :+1:

Reply:
Would it make the API more clear if a struct is expected instead of a pointer to a struct?

Reply:
I don't think switching to a struct instead of pointer makes the API any clearer. They're both nearly equivalent and there is existing convention in the code base to pass options as pointers.

Reply:
Thanks for sharing your thoughts @ashishbhate :+1: